src/reynir_correct/pattern.py

```diff
@@ -1789,7 +1772,7 @@ def create_patterns(cls) -> None:
             p.append(
                 (
                     "frétt",  # Trigger lemma for this pattern
-                    "( IP|VP ) > { NP > { 'frétt' } VP > { PP > { 'að' } } }",
+                    "( IP|VP ) > { NP > { 'frétt' } VP > { PP > { P > { 'að' } } } }",
```

Comment:
If this is actually the verb ("ég hef ekki frétt að þessu") then we should include the verb both as a trigger lemma and in the search pattern.

Reply:
This is actually used to catch "Fréttir berast af e-u", so it's not the verb.